tqdm/tests/tests_version.py

```diff
@@ -4,9 +4,10 @@
 def test_version():
     """ Test version string """
     from tqdm import __version__
-    Mmpe = re.split('[.-]', __version__)
-    assert 3 <= len(Mmpe) <= 4
+    version_parts = __version__.split()[0]  # remove part after space (branch name)
+    version_parts = re.split('[.-]', version_parts)  # split by dot and dash
+    assert 3 <= len(version_parts) <= 4
     try:
-        map(int, Mmpe[:3])
+        map(int, version_parts[:3])
     except:
```

Comment:
You want `except ValueError:`. This catches "errors" like `SystemExit` and other python internals.